mindocr/data/transforms/rec_transforms.py

```diff
@@ -400,6 +400,30 @@ def __call__(self, data):
         return data
 
 
+class Rotate90IfVertical:
+    """Rotate the image by 90 degree when the height/width ratio is larger than the given threshold.
+    Note: It needs to be called before image resize."""
+    def __init__(self, threshold: float = 2.0, direction: str = 'anti_clockwise',  **kwargs):
```

Comment:
The threshold of h/w of paddle and other part in mindocr is 1.5. Refer to: https://github.com/mindspore-lab/mindocr/blob/f399c7b1809a1df5634ddd9bdae00a6a882c5f9d/tools/infer/text/utils.py#L64